static/include/micro-rdk-hardware.md

```diff
@@ -5,6 +5,8 @@ The following ESP32 microcontrollers are supported:
 - [ESP32-WROVER Series](https://www.espressif.com/en/products/modules/esp32)
 - [ESP32-WROOM Series](https://www.espressif.com/en/products/modules/esp32) (until v0.1.7)
 
+You will also need a data cable like [this USB-C to USB-C cable](https://www.amazon.com/Amazon-Basics-Charger-480Mbps-Certified/dp/B01GGKYZQM/) to connect the microcontroller to your development machine.
```

Comment:
Interesting mine uses micro-USB. Let's maybe just keep it general
```suggestion
You will also need a data cable to connect the microcontroller to your development machine.
```

Reply:
sounds good! and yeah that's interesting, mine I'm p sure was usb-c but I might be confused as i'm not at my desk